src/web/routes.rs

```diff
@@ -129,7 +129,9 @@ pub(super) fn build_axum_routes() -> AxumRouter {
             "/favicon.ico",
             get_static(|| async { Redirect::permanent("/-/static/favicon.ico") }),
         )
-        .nest("/-/static/", build_static_router())
+        // `.nest` with fallbacks is currently broken, `.nest_service works
+        // https://github.com/tokio-rs/axum/issues/3138
+        .nest_service("/-/static", build_static_router())
```

Comment:
@Turbo87 also you might run into this one when working on crates.io :) 

( not sure how you are using nesting?)

Reply:
I think we're not using nesting and provide static assets via middleware, but we'll see...